.travis.yml

```diff
@@ -9,17 +9,11 @@ matrix:
       python: "3.5"
       env: ATOM_CHANNEL=beta
     - os: osx
-      language: generic
-      before_install:
-        - brew update
-        # Need a way to check if it's installed already or not...
-        - brew install python
```

Comment:
It seems that the OS X image in question now has `python` installed by default, where before it didn't. `brew list python` would "fail" if `python` wasn't installed, thus letting the command continue on to the `brew install python`.

Since it will now succeed as it seems they have `python` installed by default, it will amount to a simple check that it is installed before continuing.

(You can have commands run one after the other in the same "command" using `command1 && command2`, or like this you can have one run if the other fails like `failingCommand || runsWhenFailed`)